.vscode/tasks.json

```diff
@@ -0,0 +1,45 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Install AppStudio deployments",
+            "command": "./hack/bootstrap-cluster.sh",
```

Comment:
it can be run with `development` or `preview` parameter to avoid setting stage environment first.